cmd/optimize/report.go

```diff
@@ -104,6 +104,10 @@ func listReports(cmd *cobra.Command, args []string) error {
 	}
 	tempVals.WorkloadFilters = strings.Join(filtersList, " && ")
 
+	if strings.HasPrefix(tempVals.WorkloadId, "k8s:deployment:") {
```

Comment:
@linkous8 consider using strings.CutPrefix() as a simpler way to achieve the same

Reply:
Thanks for the recommendation, that's a handy function to know about